x/gov/client/utils/utils.go

```diff
@@ -22,6 +26,20 @@ func NormalizeVoteOption(option string) string {
 	}
 }
 
+// NormalizeWeightedVoteOptions - normalize normalize vote options param string
+func NormalizeWeightedVoteOptions(options string) string {
```

Comment:
would like to see a test or 2 for this function

Reply:
Added test